### ISSUE TITLE
Getting weird Windows-only failures on 0.6.0

### ISSUE THREAD

**Issue by svenstaro** [colin-kiegel/rust-pretty-assertions#29]
So I'm getting this failure that has something to do with vt100 but only on Windows: https://travis-ci.com/svenstaro/mt940-rs/jobs/179388389

I don't really understand it but it keeps me from upgrading for now. Maybe you can take a look?

**Comment by orium** [comment]
I'm having the same problem: https://travis-ci.org/orium/rpds/jobs/496248771#L187

**Comment by colin-kiegel** [comment]
Thanks for the bug report.

We introduced `vt100` dependency with PR #28. It's [source code](https://github.com/Phundrak/output-vt100-rs/blob/master/src/lib.rs) is super simple. The panic occurs in this line:
https://github.com/Phundrak/output-vt100-rs/blob/master/src/lib.rs#L36

```
thread '<unnamed>' panicked at 'assertion failed: `(left != right)`
  left: `0`,
 right: `0`', C:\Users\travis\.cargo\registry\src\github.com-1ecc6299db9ec823\output_vt100-0.1.1\src\lib.rs:36:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

**Comment by colin-kiegel** [comment]
@Phundrak can you have a look at this?
Would it be feasible to make your library more failsafe (not panicking)? E.g. by delegation of error handling `try_init() -> Result<_,_>` or something similar.

Otherwise I'd make this opt-in only for the time being.

cc @nbouteme

**Comment by nbouteme** [comment]
I think the issue is that CI systems typically redirect the executable output through a pipe and the system calls expect stdout to be a console. This also occured on AppVeyor.

Anyways as suggested, I'll add a try_init that doesn't panic.

**Comment by nbouteme** [comment]
It's done. Unfortunately I can't publish the new output_vt100 version to crates.io, as @Phundrak has yet to add me as an owner, and he hasn't been responding to my messages lately.

**Comment by colin-kiegel** [comment]
fix is out with `v0.6.1`

thx @nbouteme for your quick support :-)

**Comment by svenstaro** [comment]
Working fine, thanks guys!
